# MeshEnvironment.msg

std_msgs/Header header
string surf_path_prototype
string target_path_prototype
string full_env_path
string clustered_env_path
